src/Groups/abelian_aut.jl

```diff
@@ -0,0 +1,89 @@
+function _isomorphic_gap_group(A::GrpAbFinGen; T=PcGroup)
```

Comment:
What's the rationale for allowing an override of the group type? The cost is that we can't rely on any operations specific to the group type, we must use the most generic (and hence possible less efficient) interfaces.

In general `FpGroup` or `PcGroup` (or `PcpGroup`, to be implemented, see issue #432) are the primary candidates. It depends a bit on what we want to support here.

Reply:
OK, one reason would be because `PcGroup` does not support infinite groups, while for finite groups it may be more efficient... Hm.

Reply:
Basically, I left the choice because I do not know about the (dis)advantages of the different candidates.
I felt that it did not hurt since we can dispatch on more efficient methods in special cases?